account_move_line_stock_info/models/account_move_line.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2019 ForgeFlow S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class AccountMoveLine(models.Model):
+    _inherit = "account.move.line"
+
+    stock_move_id = fields.Many2one(
+        comodel_name="stock.move", string="Stock Move", copy=False, index=True
```

Comment:
Pplease use the same index as in https://github.com/odoo/odoo/tree/18.0/addons/stock_account/models#L10

Is this redefinition even needed?

Reply:
index updated,
and field is availble on account.move not on account.move.line

Reply:
Right. Sorry.